app/models/indicator.rb

```diff
@@ -27,9 +23,16 @@ class Indicator < ApplicationRecord
   validate :parent_is_not_variation, if: proc { |i| i.parent.present? }
   before_validation :ignore_blank_array_values
 
-  scope :system_indicators_with_variations, lambda {
-    where(parent_id: nil).
-      joins('LEFT JOIN models ON models.id = indicators.model_id').
+  pg_search_scope :search_for, against: [
+    :category, :subcategory, :name, :alias
+  ]
+
+  def self.system
```

Comment:
If we only want system indicators then `parent_id: nil, model_id: nil`

Reply:
I'm not 100% sure whether the `model_id: nil` should be included or not. It wasn't there in the original scope, that was called `system_indicators_with_variations`, so I'm confused.